ags/oidc/authz_flow.py

```diff
@@ -123,7 +124,7 @@ def auth_endpoint(self):
                 self.load_broker_config()
 
         if self._auth_endpoint:
-            return urljoin(self.broker_url, self._auth_endpoint)
+            return "%s%s" % (self.broker_url, self._auth_endpoint)
```

Comment:
I'm thinking I might extract a `build_url` method like:

```
def build_url(base_url, path):
    return '{base_url}{path}'.format(base_url=base_url, path=path)
```

What do you reckon?